realm/src/main/java/io/realm/exceptions/RealmEncryptionNotSupportedException.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2015 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.exceptions;
+
+/**
+ * On some devices (HTC One X for example), for some reason, the system doesn't pass the right
```

Comment:
Have we identified others than HTC One X? If so we just list that and mention we have only heard it for that device.